src/main/java/frc/robot/commands/autonomous/FiveCycle.java

```diff
@@ -85,7 +86,9 @@ public FiveCycle(RobotContainer robotContainer, boolean nonProcessorSide) {
 
   public Command autoHelper(PathPlannerPath pathOne, PathPlannerPath pathTwo) {
     return Commands.sequence(
-        Commands.waitSeconds(waitTime),
+        // wait until coral is loaded
+        Commands.waitUntil(() -> robotContainer.endEffector.endEffectorIO.coralLoaded()),
```

Comment:
```suggestion
        Commands.waitUntil(() -> robotContainer.endEffector.coralLoaded()),
```

With the other changes as described, the output would be this. Makes it a lot more easier to use and read.

Reply:
We actually completely refactored this to be a parameter of the command, I'm not sure how that edit didn't get commited. Thanks for pointing out the persisting issue!